tests/test_modules_properties.py

```diff
@@ -19,3 +19,18 @@ def test_basic(self):
         assert len(jjb_authorization) == 2
         assert jjb_authorization['usera'] == ['job-build', 'job-read']
         assert jjb_authorization['userb'] == ['job-configure', 'job-workspace']
+
+
+class TestRebuildSettings(object):
+    def test_basic(self):
+        filename = os.path.join(fixtures_path, 'rebuild-settings.xml')
+        root = get_xml_root(filename=filename)
+        assert root is not None
+        parent = []
+        rebuildsettings(root, parent)
+        assert len(parent) == 1
+        assert "rebuild" in parent[0]
+        rebuild = parent[0]["rebuild"]
+        assert len(rebuild) == 2
+        assert not rebuild["auto-rebuild"]
+        assert not rebuild["rebuild-disabled"]
```

Comment:
You should rather explicitly check for `False`, i.e. `assert rebuild["rebuild-disabled"] is False`, because other values can also evaluate to `False`, like an empty list `[]`, and we don't want to match them here.